### ISSUE TITLE
Test updated Annotations component for accessibility

### ISSUE THREAD

**Issue by elynema** [samvera-labs/ramp#795]
# Description

Dananji has already done some accessibility checks while going through the development process for the updated Annotations component, but it would be good to have a second set of eyes look at it. For testing we can try against WCAG 2.2 AA using existing tools to evaluate.

# Done Looks Like

 - [ ] Check for accessibility issues via Ramp demo site with SiteImprove browser plugin
 - [ ] Check for accessibility issues via Ramp demo site with Sa11y bookmarklet
 - [ ] Create new issues for any detected issues

**Comment by joncameron** [comment]
Latest WCAG: https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/

**Comment by Dananji** [comment]
I found the following error with SiteImprove:
- Role with implied hidden content has keyboard focus: This error is popping up for the annotations with links in the text. In the current implementation, annotations component is displaying these as list elements with `role='option'` which is presentational role. Relevant success criterion: https://www.w3.org/TR/WCAG/#name-role-value (Level A)

@joncameron and @elynema I didn't create a new issue for this yet, but I think this should be a simple fix to do before the release. Before we fix this issue, I think we need to discuss question on the behavior implemented in the [open PR](https://github.com/samvera-labs/ramp/pull/805) for keyboard a11y in annotations component.

**Comment by elynema** [comment]
@Dananji The ANDI bookmarklet may be easier to understand than the Sally one. Both were recommended to me at a presentation on accessibility testing tools at code4lib this year. If you cilck on the ANDI output line in the bookmarklet, it provides a little more info about how important the issue may be. Mostly everything I see in there now for the Ramp demo site are just warnings to check on things that are explicitly set as aria-hidden or explicitly have a tab-index=-1.

**Comment by Dananji** [comment]
@elynema I checked it out and yes it's easier to understand than Sally. I'm seeing a similar output when it comes to warnings. Most of the tab-index warnings can be ignored because we set `tab-index=-1` in structured navigation and transcript components to achieve the desired keyboard navigation behavior.

In addition to the current demo site, I ran ANDI with the changes in the open PR (#805), which has the latest code changes in the annotations component to implement keyboard navigation. These changes are not yet in demo site. And I was not seeing the 2 Level A alerts (mentioned above) I was seeing in other accessibility checker tools with ANDI bookmarklet.
